### ISSUE TITLE
Extend CI to do coverage testing

### ISSUE THREAD

**Issue by jitseniesen** [spyder-ide/spyder-unittest#21]

**Comment by goanpeca** [comment]
`ciocheck` command will run tests and coverage, but when tests are fixed (they are failing for me :-|) we can enforce pytest on the ciocheck config file so that test must pass to have a passing CI.

Once this is done, we just need to add a line so that when tests are finished, we upload coverage info to coveralls (I also added the badge already)

**Comment by jitseniesen** [comment]
With this issue, I actually meant that the plugin be extended so that it can display the result of coverage testing :smile: Doesn't matter though; I'll add a new issue for that.

**Comment by goanpeca** [comment]
ok ok :-p
